docs/ui/schema-conv/issues.md

```diff
@@ -32,6 +32,7 @@ Spanner migration tool scans through the generated spanner schema and notifies t
 ## Errors
 
 - Detection of unsupported check constraints in spanner.
+- Detection of unsupported default values in spanner.
```

Comment:
Will this be a warning or an error? Are we stopping the user from proceeding with the migration in case a default value isn't supported?